typedoc.json

```diff
@@ -2,8 +2,8 @@
   "$schema": "https://typedoc.org/schema.json",
   "cleanOutputDir": true,
   "entryPoints": [
-    "packages/client-core/src/index.ts",
-    "packages/client-payments/src/index.ts",
+    // "packages/client-core/src/index.ts",
```

Comment:
Are these commented out accidentally or deliberately? If deliberately, then removing the dead code altogether, with an comment in the commit message, is preferred.

Reply:
I left them bc they would be needed back eventually, but it doesn't matter anymore.